holoviews/operation/datashader.py

```diff
@@ -417,6 +418,7 @@ def _process(self, element, key=None):
                 params["vdims"] = [params["vdims"]]
             sum_agg = ds.summary(**{str(params["vdims"][0]): agg_fn, "__index__": ds.where(sel_fn)})
             agg = self._apply_datashader(dfdata, cvs_fn, sum_agg, agg_kwargs, x, y, agg_state)
+            agg.attrs["selector"] = str(sel_fn) if DATASHADER_GE_0_17_1 else None
```

Comment:
I _think_ it is ok to infer it if we don't have the latest version.

Reply:
Done in https://github.com/holoviz/holoviews/pull/6568/commits/ec4dda9a221ee20e99aa31260b1f63fa7f66f021